README.md

```diff
@@ -4,15 +4,15 @@
 
 ## Install
 
-```console
-$ npm install eslint-loader
+```bash
+$ npm install eslint-loader --save-dev
 ```
 
 ## Usage
 
 In your webpack configuration
 
-```javascript
+```js
```

Comment:
I saw that line [79 had the line](https://github.com/MoOx/eslint-loader/edit/master/README.md#L79) had that style so i confirmed the other styles to it.